src/SprykerEco/Zed/Adyen/Business/Handler/Redirect/OnlineTransferRedirectHandler.php

```diff
@@ -139,6 +139,7 @@ protected function getRequestDetails(AdyenRedirectResponseTransfer $redirectResp
     {
         return [
             AdyenApiRequestConfig::PAYLOAD_FIELD => $redirectResponseTransfer->getPayload(),
+            AdyenConfig::REDIRECT_RESULT_FIELD => $redirectResponseTransfer->getRedirectResult(),
```

Comment:
Why it's not used directly from `\SprykerEco\Shared\Adyen\AdyenApiRequestConfig::REDIRECT_RESULT_FIELD`?

Reply:
Changed to `AdyenApiRequestConfig`